src/cli/lurk_proof.rs

```diff
@@ -105,7 +105,8 @@ mod non_wasm {
                     lang,
                 } => {
                     log::info!("Loading public parameters");
-                    let pp = public_params(rc, std::sync::Arc::new(lang))?;
+                    let pp =
+                        public_params(rc, std::sync::Arc::new(lang), Some(&public_param_dir()))?;
```

Comment:
The fact that the third argument (the path) is an `Option` might indicate some plumbing that still requires work. The caller of this function must know where the public params are (or are supposed to be created).